.azure-pipelines/common2/jobs/run-pylint.yml

```diff
@@ -0,0 +1,23 @@
+parameters:
+  - name: dependsOn
+    type: object
+    default: []
+    displayName: 'Depends On'
+
+  - name: pylintConfig
+    type: string
+    default: 'pylintrc'
+    displayName: 'Pylint Configuration File'
+
+jobs:
+  - job: run_pylint
```

Comment:
Each of the job need to grab a agent pool instance and checkout code. And the job only has single small step. Is it better to use single job for these small steps?

Reply:
Yeah, I agree it could save resources. When time comes that each step takes too long and running in parallel helps, we could use jobs.